### PR TITLE
(DOCSP-14402) Externally sourced config values

### PR DESCRIPTION
**Pull request by zach-carr**
[JIra](https://jira.mongodb.org/browse/DOCSP-14402)
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/fef6357/bi-connector/docsworker-xlarge/DOCSP-14402/reference/mongosqld/#load-external-configuration-values)

Changes:
- Added a section on loading external config values with expansion directives
- Changed ToC depth to 2 because the page is pretty large with many subsections

I'm linking out to the manual because the information is identical - porting all of that info to the BI connector docs would mean maintaining it in two places. I tried to introduce the topic clearly and generally with the goal of getting users to the manual page.

### CODE REVIEW

File: source/reference/mongosqld.txt [hunk 1]
```diff
@@ -718,6 +718,17 @@ These configuration file options are also available as system variables.
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You may use expansion directives in your configuration file to load 
+externally sourced configuration values. Expansion directives help 
+obscure confidential information like security certificates and 
+passwords.
+
+To learn more about using expansion directives, see :manual:`Externally Sourced Configuration File Values </reference/expansion-directives/>` 
+in the MongoDB manual.
+
```
Comment (zach-carr) [inline source/reference/mongosqld.txt:731]: Besides this brief section, I considered a note at the very top of the Configuration File section, along the lines of:

.. note::
   You may store sensitive values outside of your configuration file 
   and load them at startup time with an automatic |api| request or 
   shell command. To learn more, see :manual:`Externally Sourced Configuration File Values </reference/expansion- 
   directives/>` in the MongoDB manual.
   
Let me know if you think that's a better way to include this info.

File: source/reference/mongosqld.txt [hunk 2]
```diff
@@ -718,6 +718,17 @@ These configuration file options are also available as system variables.
 For more information, see
 :ref:`System Variables <system-environment-variables>`.
 
+Load External Configuration Values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You may use expansion directives in your configuration file to load 
+externally sourced configuration values. Expansion directives help
```
Comment (jason-price-mongodb) [inline source/reference/mongosqld.txt:725]: This LGTM, but perhaps remove "help", or rewrite it slightly to something like "Use expansion directives to obscure..." if the directives 100% obscure the info.

File: source/reference/mongosqld.txt (hunk 1 above)
Comment (jason-price-mongodb) [reply source/reference/mongosqld.txt:731]: Perhaps put the content in an include file, and include the content twice using the include file?

**Review by jason-price-mongodb — CHANGES_REQUESTED** [review]

File: source/reference/mongosqld.txt (hunk 1 above)
Comment (zach-carr) [reply source/reference/mongosqld.txt:731]: Good idea, done

**Comment by zach-carr** [comment]
Thanks Jason! Updated here: https://docs-mongodbcom-staging.corp.mongodb.com/2b5e795/bi-connector/docsworker-xlarge/DOCSP-14402/reference/mongosqld/#configuration-file

**Pull request merged**